connectors/v2/driveplayer.com.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+/* global Connector */
+Connector.playerSelector = '.jp-play';
+
+Connector.getArtist = function () {
+	var text = $('.playing > .artist').text().trim();
+	return text || null;
+};
+
+Connector.getTrack = function () {
+	var text = $('.playing > .title').text().trim();
+	return text || null;
+};
+
+Connector.isPlaying = function () {
```

Comment:
This `isPlaying` function can be replaced to `Connector.playButtonSelector = '.jp-play';` definition. But the function is correct too, though.